packages/filesystem/src/browser/file-tree/file-tree-widget.tsx

```diff
@@ -72,16 +73,25 @@ export class FileTreeWidget extends TreeWidget {
     }
 
     protected createNodeAttributes(node: TreeNode, props: NodeProps): React.Attributes & React.HTMLAttributes<HTMLElement> {
-        const elementAttrs = super.createNodeAttributes(node, props);
```

Comment:
(fyi) it was extracted that one can see values during debugging

Reply:
done. thank you for the review !